packages/page-accounts/src/Accounts/Account.tsx

```diff
@@ -118,15 +132,24 @@ function Account ({ account: { address, meta }, className = '', delegation, filt
 
   useEffect((): void => {
     if (balancesAll) {
-      setBalance(address, balancesAll.freeBalance.add(balancesAll.reservedBalance));
+      const balance: AccountBalance = {
+        bonded: stakingInfo?.stakingLedger.active.unwrap() ?? BN_ZERO,
+        locked: balancesAll.lockedBalance,
+        redeemable: stakingInfo?.redeemable ?? BN_ZERO,
+        total: balancesAll.freeBalance.add(balancesAll.reservedBalance),
+        transferrable: balancesAll.availableBalance,
+        unbonding: calcUnbonding(stakingInfo)
+      };
+
+      setBalance(address, balance);
```

Comment:
```suggestion
      setBalance(address, {
        bonded: stakingInfo?.stakingLedger.active.unwrap() ?? BN_ZERO,
        locked: balancesAll.lockedBalance,
        redeemable: stakingInfo?.redeemable ?? BN_ZERO,
        total: balancesAll.freeBalance.add(balancesAll.reservedBalance),
        transferrable: balancesAll.availableBalance,
        unbonding: calcUnbonding(stakingInfo)
      });
```

Why not simply the above?

Reply:
:shrug: above does look better